Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Input.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import { Button } from "@material-ui/core";
+import Box from "@material-ui/core/Box";
+
+const useStyles = makeStyles((theme) => ({
+  box: {
+    margin: theme.spacing(0.5),
+    marginTop: theme.spacing(0),
+    width: 50,
+  },
+}));
+
+export default function EntryField(props) {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <Box
+        display="flex"
+        flexDirection="row"
+        p={1}
+        m={1}
+      >
+        <Box p={1}>
+          <form noValidate autoComplete="off">
+            <TextField
+              style={{ width: 500 }}
+              id="outlined-text"
+              label="Issue"
+              multiline
```

Comment:
Probably should make CSS more dynamic instead of static values like width: 500 (especially for mobile devices)